src/posts/paper/5-activity-risk.ts

```diff
@@ -122,7 +117,7 @@ _To learn about the scientific research supporting our estimates, please see the
 Now we understand Activity Risk, or how your chance of getting the virus changes based on the activity you’re doing. But Activity Risk _assumes_ the other person is COVID-positive. What are the actual chances that whoever you're interacting with has COVID? Let’s look at Person Risk to understand that.
 
 [^1]:
-     We mean surgical masks, well-fitted cloth masks with a filter, or similar high-quality masks. Bandanas, poorly-fitting masks, or DIY masks will give less protection than this.
+     The calculator and risk tracker include a more detailed breakdown of different mask types with different multipliers. The [Research Sources](14-research-sources) section explains the detailed sources and reasoning for these different types.
 
```

Comment:
We might want to capitalize Risk Tracker and make it a link.

Reply:
Oh and we might want to  link directly to the masks section

Reply:
ok!